lib/ext/heroku/command/addons.rb

```diff
@@ -225,6 +225,14 @@ def add
       end
     end
 
+    # addons:add ADDON
+    #
+    # WARNING: `heroku addons:add has been deprecated. Please use `heroku addons:create` instead.
```

Comment:
@geemus yeah I can nuke it, but it still appears in `heroku help addons`, just without any note that it's deprecated/a no-op